html-templates/people/person.tpl

```diff
@@ -13,16 +13,12 @@
             <div class="col-inner">
 
                 <header class="page-header">
-                    <div class="header-buttons">
-                    {if $.User->ID == $Person->ID || (ProfileRequestHandler::$accountLevelEditOthers && $.User->hasAccountLevel(ProfileRequestHandler::$accountLevelEditOthers))}
-                        <a class="button" href="/profile{tif $.User->ID != $Person->ID ? cat('?person=', $Person->ID)}">Edit Profile</a>
-                    {/if}
-                    </div>
-
                     <h2 class="header-title">{$Person->FullName|escape} <small class="muted">Public Feed</small></h2>
                     {if $ownProfile}
```

Comment:
only having the edit button within the `$ownProfile` check prevents admins from seeing it for other users